hare3/hare_test.go

```diff
@@ -753,7 +753,9 @@ func TestHandler(t *testing.T) {
 	})
 }
 
-func gatx(id types.ATXID, epoch types.EpochID, smesher types.NodeID, base, height uint64) types.ActivationTx {
+const epoch = types.EpochID(1)
```

Comment:
I'm not sure about this linter if enabling it ends up using globals. The flow of information is broken now and the code is more difficult to follow.